README.rst

```diff
@@ -16,12 +16,12 @@ Here's an example of usage with
 `aiohttp <https://github.com/KeepSafe/aiohttp>`_ framework:
 
 .. code-block:: python
-    
+
     app = aiohttp.web.Application()
-    aioreloader.start(app.loop)
+    aioreloader.start()
     aiohttp.web.run_app(app)
-    
+
 Requirements
 ------------
 
-Python - at least 3.4
+Python - at least 3.3
```

Comment:
https://github.com/wikibusiness/aiocassandra/blob/master/setup.py#L38

is simply to set this as not hard `install_requires`